### PR TITLE
Updated Host Information 

### LINKED ISSUE #29: [BUG]: update host information
**Describe the bug**
Jason's profile picture should be updated and Ellie needs to be added to the website.

### PR DESCRIPTION
**Pull request by DhanushNehru**
## Issue

Closes #29 

## Changes proposed 

Changed the host information accordingly

## Screenshots

Screenshot 2024-10-02 at 9 11 40 PM

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -58,6 +58,10 @@ <h2>Jason</h2>
                     <h2>Jacob</h2>
                     <a href="http://www.x.com/arcadejacob"><img class="profile" src="images/jacob.jpg" alt="Profile photo of Jacob Ashley"></a>
                 </section>
+                <section>
+                    <h2>Ellie</h2>
+                    <a href="http://www.x.com/elliezub"><img class="profile" src="images/ellie.jpg" alt="Profile photo of Ellie"></a>
+                </section>
```
Comment (shivaylamba) [inline index.html:64]: Jason Torres & Jacob Ashley - should be changed to Ellie as well

Comment (DhanushNehru) [reply index.html:64]: Do you mean there is only 1 host ? Hope I am understanding it correctly ?? @shivaylamba

Comment (DhanushNehru) [reply index.html:64]: Done @shivaylamba

**Comment by TimoCodez** [comment]
Looks good 👍

**Pull request merged**